### ISSUE TITLE
Metric list refresh only after facette restart

### ISSUE THREAD

**Issue by jerryjacobs** [facette/facette#184]
Hi all,

I run facette with collectd as provider, when collectd gets new metrics they not loaded automaticly into facette. When I restart facette then the metrics are loaded. 

Is this normal behaviour or a possible bug?
It would be nice to poll the metrics with a configuration option e.g every 30 seconds or so.

I'm running facette-0.3.0rc1-linux-amd64.tar.gz on Arch Linux

Thanks in advance,
Jerry

**Comment by falzm** [comment]
Hi Jerry

This behaviour is controllable at the provider level using the `refresh_interval` setting (documented [here](http://docs.facette.io/configuration/#toc_5)).

**Comment by jerryjacobs** [comment]
Indeed I overlooked this option, I feel a little bit ashamed that I did not fully read the documentation. 
Is there any feedback/logging when last refresh happend at provider level?

**Comment by falzm** [comment]
No problem. Periodic provider refresh is not logged, since it would pollute the logs when a short refresh interval is set.

**Comment by jerryjacobs** [comment]
You are right, for debugging this would be a nice feature when loglevel is set for providers to "verbose".
For me you may close this bug.

**Comment by falzm** [comment]
@vbatoufflet WDYT, should we add debug messages for periodic refreshes?

**Comment by vbatoufflet** [comment]
I think it could be useful when running with `debug` output level to see each refresh trigger :+1:

**Comment by jerryjacobs** [comment]
I think we should close this "bug" and open a new issue as feature request for refresh trigger logging in debugging mode.

**Comment by falzm** [comment]
See #185.
